### DISCUSSION TITLE
Temperature display error

### DISCUSSION THREAD

**Discussion by AhnJunYeongDK** [artisan-roaster-scope/artisan#939] [Q&A] [answered]
hello. My name is Junyoung Ahn and I am working as an engineer in Korea.
I have a question, so I'm writing here.

After installing the DR-25 manufactured by Diedrich Roasters, I connected the Artisan using the Modbus method and finished following the IP address and other setting methods as instructed by the head office. But there is a problem.

Since Korea is a country that uses Celsius, the DR-25 is set to Celsius.
Artisan should also be expressed in degrees Celsius. However, it is expressed in Fahrenheit.

Basically, it was set by clicking in Celsius in Artisan, and it was set to match the machine, such as the brand, machine KG, and gas type, but it is displayed in Fahrenheit due to an unknown error. How can I possibly solve this problem?

We need your opinion. Thank you for your interest.

![KakaoTalk_20220812_200453563](https://user-images.githubusercontent.com/111517312/185418164-cb01ef1b-e67c-40fb-b816-5981cc635a91.jpg)
![KakaoTalk_20220812_200453813](https://user-images.githubusercontent.com/111517312/185418174-ccd73bb3-8225-4247-ae0e-04c88b563711.jpg)
![KakaoTalk_20220812_200456160](https://user-images.githubusercontent.com/111517312/185418175-c7b73e97-7527-41bb-a168-e226947e13d6.jpg)
![KakaoTalk_20220812_201038535](https://user-images.githubusercontent.com/111517312/185418176-1980345a-9242-41cd-bdec-ed30cc0a8ef1.jpg)
![KakaoTalk_20220812_201038535_01](https://user-images.githubusercontent.com/111517312/185418179-24d8880c-8998-4352-b7aa-3953255ea2ba.jpg)
![KakaoTalk_20220812_201834390](https://user-images.githubusercontent.com/111517312/185418185-48face85-4987-44de-8661-9fd86ed475c3.jpg)

**Comment by MAKOMO** [comment, marked as answer]
That machine seems to return the temperature data in Fahrenheit. You need to indicate this to Artisan for those temperature channels such that it converts those readings automatically to the temperature units you selected for display in Artisan. Thus you need to put the mode of the first 3 MODBUS input channels to F (menu Config >> Ports; 3rd tab "MODBUS").

![diedrich-modbus](https://user-images.githubusercontent.com/13909011/185708902-11119ed5-cc82-4b3f-b178-c18f58f55a24.png)

**Comment by AhnJunYeongDK** [comment]
Thanks to you, the test was successfully completed.

https://user-images.githubusercontent.com/111517312/192366507-62cd4c79-9cf3-49cc-ba6a-0a81c7a14c52.mp4
